### PR TITLE
Add usage examples and improve documentation on tf.keras.applications models

### PR DESCRIPTION
**Pull request by Byambaa0325**
Related issue: #37996
Added some basic usage examples for models in Keras.Applications. The arguments were already added to the nightly so the changes made are:

- Add example of extracting features using premade models
- <del>Fix the issue were argument option sublist was not at the right indent level</del>
- Reworded the pooling mode argument in models' description to be more concise
- Add raises exception list where absent
- Add doctest example of creating a model and what type of model it should return

### CODE REVIEW

File: tensorflow/python/keras/applications/resnet.py [hunk 1]
```diff
@@ -544,7 +544,7 @@ def decode_predictions(preds, top=5):
       and width and height should be no smaller than 32.
       E.g. `(200, 200, 3)` would be one valid value.
     pooling: Optional pooling mode for feature extraction
-      when `include_top` is `False`.
+      when `include_top` is `False`. It could be:
       - `None` means that the output of the model will be
           the 4D tensor output of the
           last convolutional block.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/resnet.py]: I would rewrite all of these to "If `None` then the output of the model...`

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/applications/resnet.py (hunk 1 above)
Comment (Byambaa0325) [reply tensorflow/python/keras/applications/resnet.py]: That's a good idea! I was a bit hesitant to tamper with the existing documentation as I'm adding one. 
I'll rewrite it in no time.

Comment (Byambaa0325) [reply tensorflow/python/keras/applications/resnet.py]: Added a new commit changing the pooling argument description.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by Byambaa0325** [comment]
Looks like the tests are failing on doctest
" ... lacks blank after >>>: '  >>>model = DenseNet121(weights = None)'"
I'll fix it.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by Byambaa0325** [comment]
It also looks like I missed line limits a few 20 times which is failing the sanity check's pylint check.
"
> ...
> tensorflow/python/keras/applications/vgg16.py:91: [C0301(line-too-long), ] Line too long (83/80)
> tensorflow/python/keras/applications/vgg16.py:92: [C0301(line-too-long), ] Line too long (81/80)
> tensorflow/python/keras/applications/inception_resnet_v2.py:85: [C0301(line-too-long), ] Line too long (81/80)
> ...

"
Rearranging the lines going over 80 characters, and, while I'm at it, fix the grammar mistake

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**